app/Services/Blink.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Rogue\Services;
+
+use DoSomething\Gateway\Common\RestApiClient;
+
+class Blink extends RestApiClient
```

Comment:
Could we extend the `DoSomething\Gateway\Blink` class instead of creating a new one?

Reply:
That is a great idea!